### DISCUSSION TITLE
Multiple buckets sharing a single S3 access key

### DISCUSSION THREAD

**Discussion by davidparks21** [seaweedfs/seaweedfs#3773] [Q&A]
**Describe the bug**
When trying to create more than one bucket with the same S3 access key the following error is encountered:
```
error: duplicate accessKey[XXXXXXXXXXXXXXXX], already configured in user[existing-bucket-name]
```

**System Setup**
SeaweedFS 3.27

**Expected behavior**
Most S3 filesystems (AWS, Ceph, for example) allow multiple buckets per single access key. I suggest the same behavior on SeaweedFS, multiple buckets should be accessible using a single access key.

**Comment by dzadonskii** [comment]
Just define multiple buckets under same section in S3 config:

```
{
  "name": "seafile",
  "credentials": [
    {
      "accessKey": "XXXXX",
      "secretKey": "XXXXX"
    }
  ],
  "actions": [
    "Read:seafile-cob",
    "List:seafile-cob",
    "Tagging:seafile-cob",
    "Write:seafile-cob",
    "Read:seafile-fsob",
    "List:seafile-fsob",
    "Tagging:seafile-fsob",
    "Write:seafile-fsob",
    "Read:seafile-bb",
    "List:seafile-bb",
    "Tagging:seafile-bb",
    "Write:seafile-bb"
  ]
},
```
